lucet-runtime/Makefile

```diff
@@ -1,42 +0,0 @@
-# This Makefile exists primarily to allow running just the test suite for lucet-runtime, rather than
```

Comment:
Is this deleted because `cargo test -p lucet-runtime` does the same thing?

Reply:
Yeah, this should have been deleted a long time ago, I just ran into it accidentally while trying to run the C API tests this time around.